wallet/db.c

```diff
@@ -191,6 +191,8 @@ char *dbmigrations[] = {
     "ALTER TABLE payments ADD COLUMN route_channels TEXT;",
     "CREATE TABLE htlc_sigs (channelid INTEGER REFERENCES channels(id) ON DELETE CASCADE, signature BLOB);",
     "CREATE INDEX channel_idx ON htlc_sigs (channelid)",
+    /* Need to track high-watermark for wallet utxos.  */
+    "ALTER TABLE vars ADD COLUMN last_processed_block INTEGER;",
```

Comment:
This is wrong.  See line 144 for correct way to add new var. with an initial value  Note, if not added in migrations, `db_get_intvar` will return 0.

Each row in `TABLE vars` is a variable, so we should add a row, not a column.  `db_set_intvar` will automatically add a row, if not present.

Reply:
Oops, will fix, thanks!

Reply:
Hmm, I don't actually need this at all, thanks! I think I prefer all 'vars' defined in the dbmigrations array, rather than allowing them to created on the fly, but that can be done later.